tox.ini

```diff
@@ -26,36 +26,19 @@
 
 [tox]
 envlist =
-  py26,py27,py32,py33,py34,
-  py27-monolithic,py33-monolithic,py34-monolithic,
-  py27-not-relaxed-strides,py33-not-relaxed-strides,py34-not-relaxed-strides
+  py27,py34,
+  py27-not-relaxed-strides,py34-not-relaxed-strides
 
```

Comment:
Might as well have `py35` and `py36` as well. Note that you can still break the lines as this is a list.